src/main.ts

```diff
@@ -127,6 +159,10 @@ app.whenReady().then(async () => {
   watcher.on('unlink', (filePath) => {
     mainWindow.webContents.send('ui:remove-file', path.basename(filePath))
   })
+
+  splashscreenWindow.close()
+  mainWindow.show()
+  mainWindow.focus()
```

Comment:
This won't work, we will need to use the `splashscreen:close` event to make sure we show a ready page.
So we will need to detect that the react page is loaded and then we can send the event from there to trigger the hiding of the splashscreen and the showing of the main window